framework/include/base/Assembly.h

```diff
@@ -1,16 +1,13 @@
-/****************************************************************/
-/*               DO NOT MODIFY THIS HEADER                      */
-/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
-/*                                                              */
-/*           (c) 2010 Battelle Energy Alliance, LLC             */
-/*                   ALL RIGHTS RESERVED                        */
-/*                                                              */
-/*          Prepared by Battelle Energy Alliance, LLC           */
-/*            Under Contract No. DE-AC07-05ID14517              */
-/*            With the U. S. Department of Energy               */
-/*                                                              */
-/*            See COPYRIGHT for full restrictions               */
-/****************************************************************/
+/**************************************************************************************************/
+/*                                DO NOT MODIFY THIS HEADER                                       */
+/*                  MOOSE - Multiphysics Object Oriented Simulation Environment                   */
+/*                                                                                                */
+/*                    Copyrights reserved, please see COPYRIGHT for details                       */
+/*                    https://github.com/idaholab/moose/blob/master/COPYRIGHT                     */
+/*                                                                                                */
+/*                    Licensed under LGPL 2.1, please see LICENSE for details                     */
+/*                    https://www.gnu.org/licenses/lgpl-2.1.html                                  */
+/**************************************************************************************************/
```

Comment:
Why so wide?

Reply:
I think this should be as narrow as possible to fit the text.

Reply:
I think it should fit to our defined line width, it makes the right margin match the code lines.

Reply:
Honestly, I'm not a huge fan of the wide width either. I would venture that most headers aren't defined to be the width of the projects that they are on. The GPL headers are pretty short.

Reply:
So, drop the frame, align to the left and use `//` 😈

Reply:
I am with @andrsd